tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -38,6 +38,9 @@
 #include "tls/s2n_handshake_io.c"
 #include "tls/s2n_tls13_handshake.c"
 
+static int s2n_tls13_conn_copy_server_finished_hash(struct s2n_connection *conn);
+#include "tls/s2n_handshake_transcript.c"
```

Comment:
nit: swap the order so the #include is with the other #includes

Reply:
generally, keep includes together.